app/src/main/java/com/poti/android/presentation/user/withdrawal/WithdrawalScreen.kt

```
@@ -0,0 +1,89 @@
+import androidx.compose.ui.unit.dp
+import com.poti.android.R
+import com.poti.android.core.designsystem.component.button.ActionButtonType
+import com.poti.android.core.designsystem.component.button.PotiActionButton
+import com.poti.android.core.designsystem.component.display.PotiListRadio
+import com.poti.android.core.designsystem.component.navigation.PotiHeaderPage
+import com.poti.android.core.designsystem.theme.PotiTheme
+import kotlinx.collections.immutable.toImmutableList
+
+@Composable
+fun WithdrawalRoute(modifier: Modifier = Modifier) {
+}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)